node/express/containerCosmosDBWithTests/Tests/functional_tests/sampleFunctionalTests.js

```diff
@@ -26,13 +26,25 @@ describe('sampleFunctionalTests', function () {
 			});
     });
 
-    it('Assert page title', (done) => {
-		driver.get(process.env['webAppUrl']).then(() => {
-			return driver.wait(until.titleIs('Express - Node.js Express Application'), 2000);
-		})
-		.then(() => done())
-		.catch((err) => {
-			done('Failed with error ' + err);
-		});
-    });
+	it('Assert page title', async() => {
+		var numRetries = 5;
+		for (var i = 0; i < numRetries; i++)
+    	{
+			try
+			{
+				await driver.get(process.env['webAppUrl']);
+				await driver.wait(until.titleIs('Express - Node.js Express Application'), 2000);
```

Comment:
there is already a wait until 2 seconds. Would it work if we increase it to 1 min? How does wait work here?

Reply:
this is the timeout for the title to get populated once the web page is fetched. it is different than the timeout for fetching the webpage